<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ncias-q1615-v" name="TestPartnerDataUpload" tests="2" failures="2" timestamp="31 Jan 2018 21:11:31 GMT" time="84.182" errors="0">
  <testcase name="Partner_Data_Upload_Invalid_File_Format" time="48.823" classname="gov.nih.nci.TestPartnerDataUpload">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at gov.nih.nci.HelperMethods.pdfGenerator.saveImageFiles(pdfGenerator.java:302)
	at gov.nih.nci.HelperMethods.BaseTestMethods.setupAfterSuite(BaseTestMethods.java:155)
	at gov.nih.nci.TestPartnerDataUpload.Partner_Data_Upload_Invalid_File_Format(TestPartnerDataUpload.java:344)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Partner_Data_Upload_Invalid_File_Format -->
  <testcase name="Partner_Data_Upload_Valid_Data" time="35.343" classname="gov.nih.nci.TestPartnerDataUpload">
    <failure type="java.lang.AssertionError" message="ERROR: Error Launching the Web site">
      <![CDATA[java.lang.AssertionError: ERROR: Error Launching the Web site
	at org.junit.Assert.fail(Assert.java:88)
	at gov.nih.nci.HelperMethods.BaseMethods.launchSite(BaseMethods.java:90)
	at gov.nih.nci.TestPartnerDataUpload.Partner_Data_Upload_Valid_Data(TestPartnerDataUpload.java:78)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Partner_Data_Upload_Valid_Data -->
</testsuite> <!-- TestPartnerDataUpload -->
